### DISCUSSION TITLE
Inhomogeneity correction step

### DISCUSSION THREAD

**Discussion by jhuguetn** [QSMxT/QSMxT#75] [Q&A] [answered]
Hi there, 

I wonder if anyone has previous experience including the additional step for bias field correction for magnitude images before mask creation and, if that's the case, if you would recommend/suggest adding the processing step for 3T MRI data. I see it is disabled by default and -after testing adding it- its effect on final QSM images and regional magnetic susceptibility mean values seems rather minimal. 

Thoughts or suggestions are gladly welcome. 

Thanks in advance,
Jordi

**Comment by astewartau** [comment, marked as answer]
Hi @jhuguetn, thanks for opening this discussion. 

The inhomogeneity correction is disabled by default for exactly the reason you say - the effect on susceptibility, especially in 3T data, seems minimal. However, we have not investigated this systematically, and welcome any further knowledge or investigations on this topic. 

Currently, the existing masking algorithms (threshold-based using the magnitude or phase), or BET, are quite robust for most applications, and I would recommend only using the inhomogeneity correction option if there are clear problems with the bias field evident in the magnitude image (e.g. in >=7T data). This option will also have no effect on phase-based masking, unless the --add_bet option is enabled (which combines the threshold-based mask with a BET mask). 

Thanks!
Ashley

**Reply by jhuguetn** [reply]
Thanks for confirming this, Ashley. And congrats to you all for the great work put on this project.
